### PR TITLE
feat: add resource requests/limits to most applications

### PR DESCRIPTION
**Pull request by barnabasbusa**

### CODE REVIEW

**Comment by barnabasbusa** [comment]
Need to be merged first: 
https://github.com/kurtosis-tech/postgres-package/pull/14
https://github.com/kurtosis-tech/redis-package/pull/5

**Comment by barnabasbusa** [comment]
Related issue: https://github.com/kurtosis-tech/kurtosis/issues/1700

File: .github/tests/mix-with-tools-mev.yaml [hunk 1]
```diff
@@ -19,8 +19,11 @@ additional_services:
   - dora
   - prometheus_grafana
   - goomy_blob
-  - full_beaconchain_explorer
+  #- full_beaconchain_explorer
```
Comment (h4ck3rk3y) [inline .github/tests/mix-with-tools-mev.yaml:22]: lets just delete this if it doesn't need to be spun up?

File: .github/tests/mix-with-tools.yaml [hunk 2]
```diff
@@ -19,6 +19,8 @@ additional_services:
   - dora
   - prometheus_grafana
   - goomy_blob
-  - full_beaconchain_explorer
+  #- full_beaconchain_explorer
```
Comment (h4ck3rk3y) [inline .github/tests/mix-with-tools.yaml:22]: do you want to keep it commented?

File: src/mev/mev_custom_flood/mev_custom_flood_launcher.star [hunk 3]
```diff
@@ -1,6 +1,12 @@
 PYTHON_IMAGE = "ethpandaops/python-web3"
 CUSTOM_FLOOD_SERVICE_NAME = "mev-custom-flood"
 
+# The min/max CPU/memory that mev-custom-flood can use
+MIN_CPU = 10
```
Comment (h4ck3rk3y) [inline src/mev/mev_custom_flood/mev_custom_flood_launcher.star:5]: where are all of these numbers coming from?

File: src/package_io/input_parser.star [hunk 4]
```diff
@@ -34,7 +34,7 @@ HIGH_DENEB_VALUE_FORK_VERKLE = 20000
 
 # MEV Params
 FLASHBOTS_MEV_BOOST_PORT = 18550
-MEV_BOOST_SERVICE_NAME_PREFIX = "mev-boost-"
+MEV_BOOST_SERVICE_NAME_PREFIX = "mev-boost"
```
Comment (h4ck3rk3y) [inline src/package_io/input_parser.star:37]: do we not need the extra `-`?

**Review by h4ck3rk3y — APPROVED** [review]
Where are all of these numbers coming from?

File: src/package_io/input_parser.star (hunk 4 above)
Comment (barnabasbusa) [reply src/package_io/input_parser.star:37]: added EL_CL name to the mev boost, see below,

File: src/mev/mev_custom_flood/mev_custom_flood_launcher.star (hunk 3 above)
Comment (barnabasbusa) [reply src/mev/mev_custom_flood/mev_custom_flood_launcher.star:5]: yolo

**Pull request merged**